prow-configurations/README.md

```diff
@@ -31,7 +41,17 @@ The installation script performs the following steps to install Prow:
 - Create secrets for HMAC and OAuth2 to be used by Prow.
 - Deploy Prow components with the `a202e595a33ac92ab503f913f2d710efabd3de21`revision.
 - Add annotations for the Prow Ingress to make it work with the NGINX Ingress Controller.
+<!--- 
+TODO not implemented yet:
 - Upload the set of configurations for plugins.
+-->
+
+To check if installation was successful, perform following steps:
+- check if all pods are up and running:
+```kubeclt get pods```
+- check if deck is accessible from outside of the cluster
+```kubectl get ing ing```
+Copy address and open it in a browser. Dashboard with Prow status should be displayed.
```

Comment:
`3. Copy the address and open it in a browser to display the Prow status on the dashboard.`